scripts/lib/install

```diff
@@ -99,9 +99,19 @@ export LANG="en_US.UTF-8"
 export LANGUAGE="en_US.UTF-8"
 
 # Check for a supported OS release.
-apt-get install -y lsb-release
-os_info="$(lsb_release --short --id --release --codename)"
-{ read -r os_id; read -r os_release; read -r os_codename; } <<< "$os_info"
+if [ -f /etc/debian_version ]; then
+    apt-get install -y lsb-release
+    os_info="$(lsb_release --short --id --release --codename)"
+    { read -r os_id; read -r os_release; read -r os_codename; } <<< "$os_info"
+elif [ -f /etc/os-release ]; then
+    source /etc/os-release
```

Comment:
`source`ing this file may unpredictably pollute our variable namespace. It would be better to confine this polution to a subshell:

```bash
os_info="$(. /etc/os-release; printf '%s\n' "$ID" "$VERSION_ID")"
{ read -r os_id; read -r os_version_id; } <<< "$os_info"
```

Reply:
Fixed.